src/components/SharedComponents/QualityGradeStatus/QualityGradeStatus.js

```diff
@@ -26,24 +24,31 @@ const qualityGradeSVG = ( qualityGrade, color ) => {
     return (
     // $FlowIgnore
       <NeedsIdGrade
-        accessible
-        accessibilityLabel={t( "Quality-Grade-needs_id" )}
         color={color}
       />
     );
   }
   return (
     // $FlowIgnore
     <CasualGrade
-      accessible
-      accessibilityLabel={t( "Quality-Grade-casual" )}
       color={color}
     />
   );
 };
 
-const QualityGradeStatus = ( { qualityGrade, color }: Props ): React.Node => (
-  <View>{qualityGradeSVG( qualityGrade, color )}</View>
-);
+const QualityGradeStatus = ( { qualityGrade, color }: Props ): React.Node => {
+  const setAccessibilityLabel = () => (
+    ( qualityGrade ) ? `Quality-Grade-${qualityGrade}` : "No grade" );
+
+  return (
+    <Pressable
```

Comment:
Why did you change to Pressable here, if this button has no function when pressed it can be a plain View?